src/ContaoCoreBundle.php

```diff
@@ -47,10 +46,8 @@ public function boot()
      */
     public function build(ContainerBuilder $container)
     {
-        $container->addCompilerPass(new AddContaoResourcesPass($this->getPath() . '/Resources/contao'));
+        parent::build($container);
```

Comment:
We don't know if that will change. I followed the Symfony example here: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php#L66